src/Emit.hs

```diff
@@ -562,7 +564,7 @@ toC toCMode (Binder meta root) = emitterSrc (execState (visit startingIndent roo
               do argListAsC <- createArgList indent (mode == ExternalCode) args
                  let Just (FuncTy _ retTy _) = ty func
                      funcToCall = pathToC path
-                 if retTy == UnitTy
+                 if not (notUnit retTy)
```

Comment:
I guess it could be a separate PR, but double negation always takes me a while to grok. 

This could just be `isUnit retTy` and filters could use `not . isUnit`

Reply:
Very much agree! I’ll merge this now, but let’s make that change.